packages/parser/src/parser.ts

```diff
@@ -115,3 +115,4 @@ export function parseForESLint(
 }
 
 export { ParserServices, ParserOptions };
+export { clearCaches } from '@typescript-eslint/typescript-estree';
```

Comment:
I've exported this here so that external people can clear the caches in their tests if they need to.
I found that without clearing the caches, the tests were failing on the CI (linux), because it the OS would run out of watch handles.

They shouldn't need this unless they're using the parser without our custom `RuleTester` though.